File: app/src/main/java/com/joshuahalvorson/safeyoutube/database/PlaylistDatabase.java
package com.joshuahalvorson.safeyoutube.database;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

@Database(entities = {Playlist.class}, version = 1)
public abstract class PlaylistDatabase extends RoomDatabase {
    public abstract PlaylistDao playlistDao();
}

File: app/src/main/java/com/joshuahalvorson/locklist/database/PlaylistDatabase.kt
package com.joshuahalvorson.locklist.database

import androidx.room.Database
import androidx.room.RoomDatabase

@Database(entities = [RemotePlaylist::class, LocalPlaylist::class], version = 1)
abstract class PlaylistDatabase : RoomDatabase() {
    abstract fun remotePlaylistDao(): RemotePlaylistDao
    abstract fun localPlaylistDao(): LocalPlaylistDao
}